github/Requester.py

```diff
@@ -251,23 +251,34 @@ def __init__(self, login_or_token, password, base_url, timeout, client_id, clien
         self.__userAgent = user_agent
         self.__apiPreview = api_preview
 
-    def requestJsonAndCheck(self, verb, url, parameters=None, headers=None, input=None, cnx=None):
-        return self.__check(*self.requestJson(verb, url, parameters, headers, input, cnx))
+    def requestJsonAndCheck(self, verb, url, parameters=None, headers=None, input=None):
```

Comment:
There are still some usage of `requestJsonAndCheck()` that uses the `cnx` argument, you might wanna refactor those as well.

Reply:
I checked the source code but I can't find them anywhere else that I didn't refactor yet (and that was the `status.github.com` stuff)